### PR TITLE
Added timestampToDate builtin function and modified toInt in order to…

### PR DESCRIPTION
**Pull request by SergioVelascoHernandez**
... process correctly the numbers in int or float format

### CODE REVIEW

File: RecordMapper/appliers/NestedSchemaSelectorApplier.py [hunk 1]
```diff
@@ -66,7 +66,7 @@ def select_nested_schema_and_add_their_fields(self, record: dict, flat_schema: d
             if field_data.selector != None and field_data.selector(field_key, record, self.custom_variables) != None else None
 
         # Not schema selected (this field will be null)
-        if nested_schema_name == None:
+        if nested_schema_name is None:
```
Comment (nradz) [inline RecordMapper/appliers/NestedSchemaSelectorApplier.py:69]: why? :thinking:

Comment (nradz) [reply RecordMapper/appliers/NestedSchemaSelectorApplier.py:69]: just kidding :tongue:

File: RecordMapper/builders/BuiltinFunctions.py [hunk 2]
```diff
@@ -19,19 +20,27 @@ def toNull():
...
+    def transform_function(current_value: object, record: dict, complete_transform_schema: dict,
+                           custom_variables: dict):
         value_to_return = None
+        
         if current_value is not None:
-            if str(current_value).isdigit():
-                value_to_return = int(round(float(current_value)))
+            try:
+                float_current_value = float(current_value)
+                value_to_return = int(round(float_current_value))
```
Comment (nradz) [inline RecordMapper/builders/BuiltinFunctions.py:41]: I think that this has to be "math.floor" to remove decimal values and avoid "round" behaviour.

File: RecordMapper/builders/BuiltinFunctions.py [hunk 3]
```diff
@@ -85,16 +94,36 @@ def transform_function(current_value: object, record: dict, complete_transform_s
...
+    """
+
+    def transform_function(current_value: object, record: dict, complete_transform_schema: dict, custom_variables: dict,
+                           is_nested_record: bool = False):
+        if current_value is None:
+            return None
+        else:
+            dt = datetime.fromtimestamp(float(current_value) / 1000)
+
+            formatted_time = dt.strftime(f'{format}')[:-3]
```
Comment (nradz) [inline RecordMapper/builders/BuiltinFunctions.py:111]: This "[:-3]" is very confusing. Add some comment!

**Review by nradz — APPROVED** [review]

File: RecordMapper/appliers/NestedSchemaSelectorApplier.py (hunk 1 above)
Comment (SergioVelascoHernandez) [reply RecordMapper/appliers/NestedSchemaSelectorApplier.py:69]: F**k, it happened again, sorry!, now it is fixed (again)

File: RecordMapper/builders/BuiltinFunctions.py (hunk 3 above)
Comment (SergioVelascoHernandez) [reply RecordMapper/builders/BuiltinFunctions.py:134]: In fact, it is not needed as the format is passed to the function as parameter.

**Pull request merged**